### ISSUE TITLE
Main process save doesn't restart the app

### ISSUE THREAD

**Issue by hosnar** [sprout2000/create-electron-webpack#8]
When saving some change in main.js file it says "[electronmon] renderer file change: src/main.js" but it does not restart the app / main process. What am i doing wrong?

**Comment by Inalegwu** [comment]
Have you checked your src/ folder for duplicate files with the .js extension ?

**Comment by hosnar** [comment]
Yes, i also tried to create fresh project and ran it there, same result.

**Comment by sprout2000** [comment]
@hosnar 
I also confirmed that the main process change does not make the app restart. Sorry for the inconvenience.
I am currently investigating, but I think it is probably a compatibility issue between [electronmon](https://github.com/catdad/electronmon) and the OS.

- macOS Sonoma 14.2.1
- React x TypeScript

Unfortunately, the last release of electronmon was 3 years ago, so a solution would be difficult.
